localisation/00_zuwc_united_localisation.csv

```diff
@@ -2582,20 +2582,20 @@ remove_5_mana_effect_tooltip;
 current_fel_corruption_tooltip;?urrent �Y[This.fel_corruption.GetName]�! is �Y[This.fel_corruption.GetValue]�!\n;;;;;;;;;;;;;x
 current_void_corruption_tooltip;?urrent �Y[This.void_corruption.GetName]�! is �Y[This.void_corruption.GetValue]�!\n;;;;;;;;;;;;;x
 pirate_culture_opinion;[pirate_culture.GetName] Opinion;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+can_be_feudal_trigger_tooltip;Civilized religion or culture\n;;;;;;;;;;;;;x
+create_religious_head_effect_tooltip;Invites character to be your religious head, [Root.GetRelHeadTitle]\n;;;;;;;;;;;;;x
+create_religious_head;Establish [Root.GetRelHeadTitle];;;;;;;;;;;;;x
```

Comment:
Unsure about how it may appear in the tooltip, e.g. "Establish High Shaman". 
```suggestion
create_religious_head;Establish the office of [Root.GetRelHeadTitle];;;;;;;;;;;;;x
```